### PR TITLE
Added slideshow buttons

### PR DESCRIPTION
**Pull request by IzabelaBakollari**
Added previous and next buttons to the slideshow. Also onclick the slideshow stops. This PR is related to the #895 issue. Still draft as it is a proposal for now. Your feedback needed :)

screenshot_20190308_163124

### CODE REVIEW

File: src/gui/wizard/slideshow.h [hunk 1]
```diff
@@ -24,7 +24,7 @@ namespace OCC {
 
 /**
  * @brief The SlideShow class
- * @ingroup gui
+ * @ingrouep gui
```
Comment (camilasan) [inline src/gui/wizard/slideshow.h]: I guess this was a typo :-) Please, undo it.

File: mirall.desktop.in [hunk 2]
```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env xdg-open
```
Comment (camilasan) [inline mirall.desktop.in]: ?

**Comment by jancborchardt** [comment]
Good stuff @IzabelaBakollari! :)

Some design feedback:
- Could you use icons for the buttons: [view-previous.svg](https://github.com/nextcloud/server/blob/master/core/img/actions/view-previous.svg) and [view-next.svg](https://github.com/nextcloud/server/blob/master/core/img/actions/view-next.svg)
- The button outline should ideally not be visible.
- Is it possible to also reduce the opacity to 30% (because if the icons are white they are too present compared to the rest).

See this mockup:
connection slideshow


What do you think? :)

**Comment by jancborchardt** [comment]
(Also, there’s already a next good issue with the visibility of the "Host your own server" on dark theme. :) – Should be 50% grey probably.)

**Comment by IzabelaBakollari** [comment]
@jancborchardt thank you for your feedback!

If I use a bright icon would not be visible on the white theme. What colour do you think suits both themes for the navigation buttons? I would say the same blue from the images of the slider. 

About "Host your own server" link, it is the same colour on both themes and changing it to grey would hide it on the white one. The current colour is #0000ff.

File: mirall.desktop.in (hunk 2 above)
Comment (IzabelaBakollari) [reply mirall.desktop.in]: I did this by accident. Sorry, will undo it. :)

**Comment by jancborchardt** [comment]
> If I use a bright icon would not be visible on the white theme. 

The icons actually have integrated shadow so they will work on both light and dark backgrounds. :)

> About "Host your own server" link, it is the same colour on both themes and changing it to grey would hide it on the white one. 

Same here, if we use a grey like for example `#888888` it will work on light as well as dark backgrounds.

See this mockup – all the same icon (view-next.svg) and the same text color (`#888888`) :slightly_smiling_face: 
In both cases we don’t use any color, because that would take attention away from the main important element, which is the Register and Log in button. :)

desktop view

**Comment by IzabelaBakollari** [comment]
I have made the changes and that's how it looks. I dont know why I cannot push on the code from my forked repository.

Screenshot_20190318_135458

**Comment by camilasan** [comment]
You need to rebase @IzabelaBakollari ;-)

**Comment by IzabelaBakollari** [comment]
I opened a new pull request just to have it more clean and understandable. 

https://github.com/nextcloud/desktop/pull/1149

**Pull request closed without merging**